lib/puppet/file_system/windows.rb

```diff
@@ -4,6 +4,8 @@
 class Puppet::FileSystem::Windows < Puppet::FileSystem::Posix
   FULL_CONTROL = Puppet::Util::Windows::File::FILE_ALL_ACCESS
   FILE_READ = Puppet::Util::Windows::File::FILE_GENERIC_READ
+  FILE_WRITE = Puppet::Util::Windows::File::FILE_GENERIC_WRITE
+  FILE_RW = (FILE_READ | FILE_WRITE)
```

Comment:
cosmetic, it could be FILE_READ_WRITE to be consistent with FILE_READ and FILE_WRITE